app/jsx/pages/BrowsePage.jsx

```diff
@@ -7,7 +7,7 @@ import PageBase from './PageBase.jsx'
 import Header1Comp from '../components/Header1Comp.jsx'
 import Header2Comp from '../components/Header2Comp.jsx'
 import Subheader2Comp from '../components/Subheader2Comp.jsx'
-import Nav1Comp from '../components/Nav1Comp.jsx'
+import NavComp from '../components/NavComp.jsx'
```

Comment:
Will plan on removing all references to Nav1Comp and Nav2Comp. This is just a start.